app/views/layouts/_user_avatar.erb

```diff
@@ -11,5 +11,5 @@
         <%= classes %>
         "
 >
-  <%= user.name.initials&.upcase %>
+  <%= only_user_initials(user.name.initials).upcase %>
```

Comment:
Also, I think I’d keep the &.upcase rather than removing &, at least the method the way I wrote it returns a nil if it’s passed a nil, which seems like reasonable behavior

Reply:
&.upcase on the method.

Reply:
Will do.